### PR TITLE
fixed dummy writer

### PR DESCRIPTION
**Pull request by bupy7**
This is fix of https://github.com/doctrine/DoctrineModule/issues/603

### CODE REVIEW

File: src/DoctrineORMModule/Service/MigrationsConfigurationFactory.php [hunk 1]
```diff
@@ -45,7 +47,13 @@ public function __invoke(ContainerInterface $container, $requestedName, array $o
         $connection       = $container->get('doctrine.connection.' . $name);
         $appConfig        = $container->get('config');
         $migrationsConfig = $appConfig['doctrine']['migrations_configuration'][$name];
-        $configuration    = new Configuration($connection);
+
+        $output = new ConsoleOutput;
```
Comment (TomHAnderson) [inline src/DoctrineORMModule/Service/MigrationsConfigurationFactory.php]: Should be `$output = new ConsoleOutput();`

**Comment by TomHAnderson** [comment]
This looks like a good fix.  Please fix the one comment I had and I'll merge asap.

**Comment by bupy7** [comment]
@TomHAnderson Done.

**Comment by TomHAnderson** [comment]
Fixes #290

**Pull request merged**